ui/src/app/pages/admin/admin-user-common.tsx

```diff
@@ -174,6 +175,22 @@ export const displayModuleExpirationDate = (
     '-'
   );
 
+export const displayTierBadge = (
```

Comment:
I'd prefer a different name here, because this function is specifically for when a module is required for a tier.  

This name does not imply anything about requirements to me.

Reply:
ok